tests/src/core/Polyfill.test.ts

```
@@ -35,4 +35,39 @@ describe("Polyfill", () => {
     });
   });
+
+  it("AudioContext polyfill", async () => {
+    delete window.AudioContext;
+      
+    (window as any).webkitAudioContext = class MockWebkitAudioContext {
+      state = "suspended";
+      
+      constructor() { }
+      
+      decodeAudioData(arrayBuffer: ArrayBuffer, successCallback: Function) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)